app/models/unavailable_as_candidate_for.rb

```diff
@@ -10,6 +10,7 @@
 class UnavailableAsCandidateFor < ActiveRecord::Base
   belongs_to :client
   belongs_to :route, class_name: 'MatchRoutes::Base', primary_key: :type, foreign_key: :match_route_type
+  has_paper_trail
```

Comment:
Is this just to capture the whodunnit? Doesn't seem like there is going to be any history except creation and deletion events...

Reply:
Yes, this is just so we can say when a client was unavailable (since that can now be removed).  And to capture who made them available when they match something they shouldn't have.